bai2/constants.py

```diff
@@ -527,6 +529,8 @@ class TypeCodeLevel(Enum):
     TypeCode('728', TypeCodeTransaction.credit, TypeCodeLevel.detail, 'Amount Applied to Service Charge'),
     TypeCode('760', TypeCodeTransaction.debit, TypeCodeLevel.summary, 'Loan Disbursement'),
     TypeCode('890', TypeCodeTransaction.misc, TypeCodeLevel.detail, 'Contains Non-monetary Information'),
+    TypeCode('906', TypeCodeTransaction.misc, TypeCodeLevel.detail, 'Today’s Opening 1 Day Float'),
+    TypeCode('907', TypeCodeTransaction.misc, TypeCodeLevel.detail, 'Today’s Opening 2+ Day Float'),
```

Comment:
I think these should have `None` in the second position rather than a `TypeCodeTransaction` value.